### DISCUSSION TITLE
Gem is out of date?

### DISCUSSION THREAD

**Discussion by BlackPhoenix** [timdorr/tesla-api#368] [General]
When I "gem install tesla_api", I get the version that still sends the user-agent (and thus does not work). Is there something I'm going wrong?

**Comment by timdorr** [comment]
I haven't published the latest stuff on master. I'm waiting to see if the community has settled on the "right" way to handle headers, as they are still somewhat in flux. I don't want to push a release that breaks things for everyone worse than it already is. 

For now, you can pass `client_options` to `Tesla::Client` to override the user-agent header.
